lib/utils/ssh.ts

```diff
@@ -162,6 +162,7 @@ function sshErrorMessage(exitSignal?: string, exitCode?: number) {
 				msg.push(`
 Are the SSH keys correctly configured in balenaCloud? See:
 https://www.balena.io/docs/learn/manage/ssh-access/#add-an-ssh-key-to-balenacloud`);
+				msg.push('Are you using `sudo` by mistake?');
```

Comment:
`sudo` doesn't usually change the homedir / `$HOME` env var, right? -
```
 $ /bin/bash -c 'echo $HOME'
/Users/paulo

$ sudo /bin/bash -c 'echo $HOME'
Password:
/Users/paulo
```
But logging in as `root` does:
```
$ sudo su -
Paulos-MBP:~ root# echo $HOME
/var/root
```
So perhaps:
```
msg.push('Are you accidentally running under the `root` user account?');
```

Reply:
(I like `accidentally` vs `by mistake` though)